src/Fhir.Anonymizer.Shared.Core/Processors/Settings/PerturbSetting.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using EnsureThat;
+using Fhir.Anonymizer.Core.AnonymizerConfigurations;
+
+namespace Fhir.Anonymizer.Core.Processors.Settings
+{
+    public class PerturbSetting
+    {
+        public double Span { get; set; }
```

Comment:
Should we add PerturbSetting's tests just like that for SubstituteSetting?

Reply:
Sure. I have added the tests. Thanks